common/src/main/java/com/microsoft/identity/common/internal/dto/Account.java

```diff
@@ -392,28 +392,42 @@ public void setAvatarUrl(final String avatarUrl) {
     //CHECKSTYLE:OFF
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (this == o){
```

Comment:
Please suppress this PMD warning. The equals() is auto-generated by Android Studio and will be regenerated each and every time the properties of these classes change.

Reply:
+1

Reply:
Let's revert these changes too unless there's a bug or anything being fixed here. I'd prefer us not to invest cycles reformatting generated code

Reply:
Reverted the changes, just supressing PMD here